mlflow/keras/load.py

```diff
@@ -45,7 +45,8 @@ def predict(self, data, **kwargs):
                 f"received type: {type(data)}.",
                 INVALID_PARAMETER_VALUE,
             )
-        return model_call(data)
+        # Return value must be json serializable, so we convert the Tensor output to a list.
+        return keras.ops.convert_to_numpy(model_call(data)).tolist()
```

Comment:
do we need to convert to list? Can we return a numpy array?

Reply:
I think numpy array is not directly serializable?

Reply:
I think numpy array works, as `_get_jsonable_obj` will convert it to list anyways